### DISCUSSION TITLE
Lists of multiple servers and quick switching between servers

### DISCUSSION THREAD

**Discussion by OmlineEditor** [rustdesk/rustdesk#8622] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I have several people who use their servers for work. I have access to several servers at the same time. Please make it convenient to change the IP of the server and repeater.

**Describe the solution you'd like**
Add the use of a list of servers so that you can quickly switch from the main window.

**Describe alternatives you've considered**
Filling out fields manually, and this is very inconvenient

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/1258
